index.html

```diff
@@ -7,7 +7,6 @@
 <body>
 
```

Comment:
- [ ] As you were working with [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md), kindly follow that flow according to the rules. This Pull Request should be from a feature to the main branch. 💯 
- [ ] As Linters are really useful to avoid mistakes before doing a Pull Request or merging a branch, they are mandatory for the whole program on Microverse. Please make sure that your linters are working when you create a Pull Request. [Here](https://github.com/microverseinc/linters-config) You can follow the instructions to install them (HTML and CSS only for now) 🥇 
- [ ] Kindly when you create a Pull Request, it's better to add a description about what are you implementing on this PR. You can follow the requirements as a description on it.

Reply:
I have deleted all the comments... 
sorry I forgot about the messy ones. Can you please review the code once again ?